### PR TITLE
WIP: feat(css prefix): add flex support

### PR DESCRIPTION
**Pull request by harxki**
## Summary
Part of #1423

## Test Plan
`./rome ci`

### CODE REVIEW

File: internal/compiler/transforms/compile/test-fixtures/css-handler/prefix/flex/input.test.md [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+
+### `Output`
+
+```css
+.style {
+	-moz-box-shrink: 1;
+	-webkit-flex-shrink: 1;
+	flex-shrink: 1;
+}
+
```
Comment (harxki) [inline internal/compiler/transforms/compile/test-fixtures/css-handler/prefix/flex/input.test.md:74]: HI! I have a questions.

There is a slight difference when compared to the output of [Autoprefixer](https://autoprefixer.github.io/).
The Autoprefixers://autoprefixer.github.io/) output does not have `-moz-box-shrink`.
Is this normal behavior?
Also, there are other `moz-*` that Autoprefixer doesn't output.

Comment (UsamaHameed) [reply internal/compiler/transforms/compile/test-fixtures/css-handler/prefix/flex/input.test.md:74]: @harxki did you set BrowsersList to `>0%`?

Comment (harxki) [reply internal/compiler/transforms/compile/test-fixtures/css-handler/prefix/flex/input.test.md:74]: thanks for the reply.
yes. Browserslist is set to `>0%`

Comment (harxki) [reply internal/compiler/transforms/compile/test-fixtures/css-handler/prefix/flex/input.test.md:74]: @jer3m01 
Should I write about these behaviors as an issue?

Comment (ematipico) [reply internal/compiler/transforms/compile/test-fixtures/css-handler/prefix/flex/input.test.md:74]: There are a bunch of tasks that @jer3m01 opened to track these works. He's aware of them :)

**Comment by jamiebuilds** [comment]
Hi! Thank you so much for your contribution!

We recently made the decision to [migrate the Rome codebase to Rust](https://rome.tools/blog/2021/09/21/rome-will-be-rewritten-in-rust) and making some significant changes to its architecture.

We're going through and closing pull requests and issues that were specific to the TypeScript codebase, including this one. But while we're closing this, we'd love to have your involvement in the future.

Thanks again!

**Pull request closed without merging**